addons/delivery/models/stock_move.py

```diff
@@ -2,12 +2,18 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from odoo import api, fields, models
-
+from odoo.tools.sql import column_exists, create_column
 
 
 class StockMove(models.Model):
     _inherit = 'stock.move'
 
+    def _auto_init(self):
+        if not column_exists(self.env.cr, "stock_move", "weight"):
+            # By creating the column 'weight' manually we steer clear of hefty data computation.
+            create_column(self.env.cr, "stock_move", "weight", "NUMERIC")
```

Comment:
I would do the computation also. like we do in 
https://github.com/odoo/odoo/blob/4e85ee767f9cc0781823c0bfb4c79df8da1c6c2e/addons/mrp/__init__.py#L23-L24
I fear the compute not retriggered later.